sd_source/SimpleDesk-AdminMaint.php

```diff
@@ -756,7 +756,7 @@ function shd_maint_starter_updater()
 				LEFT JOIN {db_prefix}helpdesk_ticket_replies AS hdtr_first ON (hdt.id_first_msg = hdtr_first.id_msg)
 				LEFT JOIN {db_prefix}helpdesk_ticket_replies AS hdtr_last ON (hdt.id_last_msg = hdtr_last.id_msg)
 			WHERE hdt.id_ticket IN ({array_int:tickets})
-			GROUP BY hdt.id_ticket',
+			GROUP BY 1, 2, 3',
```

Comment:
What benefit does this have?

Reply:
well the query in this stat is wrong...
i hat two option:
- remove the group by
- group all col

i take the last option
group by 1,2,3 means group by column1,column2 and column3 (what ever ther name are)